docs/docs/developers/setup/index.md

```diff
@@ -1,46 +1,64 @@
 
-To initialize the project Maven archetype will be used. To continue you should have Maven installed. Check your Maven version. 
 
-``` script
+## Prerequisites
+To create and run a project, you will need [Java 17](https://adoptium.net/) or later, [Maven](https://maven.apache.org/) 3.8.5 or later and [Docker Desktop](https://www.docker.com/products/docker-desktop/).
+
+You can check your Java version in most environments with:
+
+```
+java -version
+openjdk 17.0.2 2022-01-18
+OpenJDK Runtime Environment Temurin-17.0.2+8 (build 17.0.2+8)
+OpenJDK 64-Bit Server VM Temurin-17.0.2+8 (build 17.0.2+8, mixed mode)
+```
+
+Check your Maven version with:
+
+```
 mvn --version
 Apache Maven 3.8.5 (3599d3414f046de2324203b78ddcf9b5e4388aa0)
-...
 ```
 
-System requirements: Java 17, Maven 3.8.5 +
+Check your Docker installation by running:
 
-Open the command line at an empty folder and generate a project.
+```
+docker --version
+Docker version 20.10.14
+```
 
-Use _groupId_, _artifactId_, and version params to declare Maven artifacts for your project.
-Use _projectName_, _projectId_ and package to define your project name used in UI, id used for technical needs (application resources paths, Docker images names), and root package for Java code.
+## Generate a project
+To create a new project, first open a command prompt and cd to the directory where you want to store your project.
 
-Use _archetypeVersion_ parameter to set the archetype version you want to use.
-You can check the latest version [here](https://search.maven.org/search?q=g:pl.ds.websight%20a:websight-cms-ce-project-archetype).
+Next, use a Maven command to create your project, being sure to declare the desired parameters properly.
 
-Use _cmsVersion_ parameter to set WebSight CMS version used by the generated project.
-You can check the latest version in the [Release Notes](../../release-notes).
+- Use the _groupId_, _artifactId_, and version parameters to declare Maven artifacts for your project. 
+- Use _projectName_, _projectId_ and _package_ parameters to define technical requirements (such as application resources paths and Docker image names) and the root package for Java code.
+- Use the _archetypeVersion_ parameter to set the archetype version you want to use. You can check the latest version [here](https://search.maven.org/search?q=g:pl.ds.websight%20a:websight-cms-ce-project-archetype).
+- Use the _cmsVersion_ parameter to specify the WebSight CMS version that the project should use. Check the WebSight [Release Notes](../../release-notes/) to find the latest available version.
 
 Example command:
-``` script
+
+```
 mvn archetype:generate                                     \
   -DarchetypeGroupId=pl.ds.websight                        \
   -DarchetypeArtifactId=websight-cms-ce-project-archetype  \
-  -DarchetypeVersion=1.3                                   \
+  -DarchetypeVersion=1.2                                   \
```

Comment:
don't downgrade the archetypeVersion and cmsVersion (it was bumped in the meantime)